local-config-sample.php

```diff
@@ -7,23 +7,63 @@
  *
  * The default WordPress databases defines are automatically populated in
  * dev/docker/docker-compose.yml which you can override here with custom
- * defines e.g. define( 'DB_NAME', 'tribe_square1' );
-*/
+ * defines, e.g. define( 'DB_NAME', 'tribe_square1' ). If you are using Local,
+ * you will need to define these here.
+ */
+//define( 'DB_NAME', 'tribe_square1' );
+//define( 'DB_USER', 'root' );
+//define( 'DB_PASSWORD', 'root' );
+//define( 'DB_HOST', 'localhost' );
+//define( 'DB_TABLE_PREFIX', 'wp_' );
+
+/*
+ * Multisite
+ *
+ * If you enable multisite in wp-config.php, ensure to provide your local.tribe URI here
+ */
+//define( 'DOMAIN_CURRENT_SITE', 'square1.tribe' );
+
+/*
+ * Debugging
+ */
+define( 'WP_DEBUG', true );
+define( 'WP_DEBUG_LOG', true );
+define( 'WP_DEBUG_DISPLAY', true );
+define( 'SAVEQUERIES', true );
+define( 'SCRIPT_DEBUG', true );
+define( 'WP_CACHE', false );
```

Comment:
Would it be cleaner to set the [environment variable](https://make.wordpress.org/core/2020/08/27/wordpress-environment-types/) to `local` and let [wp-config-environment.php](https://github.com/moderntribe/square-one/blob/e410ce26ba03796d914896e65564d5dab6c0009d/wp-config-environment.php#L158) handle these in the config_defaults?

Reply:
I agree. We have some deployments where we us the wp-config-local as a template for other environments, so I'd rather key off `WP_ENVIRONMENT_TYPE`.

Reply:
I'll just leave this here... https://github.com/moderntribe/square-one/pull/703#issuecomment-827605757
```php
/**
 * Set the current environment type. Accepted values:
 * - production (default)
 * - staging
 * - development
 * - local
 *
 * @link https://make.wordpress.org/core/2020/07/24/new-wp_get_environment_type-function-in-wordpress-5-5/
 */
define( 'WP_ENVIRONMENT_TYPE', 'local' );

// Fallback for older environments.
define ( 'ENVIRONMENT', 'LOCAL' );
```

Reply:
Perfect 😂